projects/js-packages/licensing/components/activation-screen-error/index.tsx

```diff
@@ -0,0 +1,53 @@
+import jetpackAnalytics from '@automattic/jetpack-analytics';
+import { Icon, warning, check } from '@wordpress/icons';
+import React, { FC, useEffect } from 'react';
+import { LICENSE_ERRORS } from './constants';
+import { UseGetErrorContent } from './use-get-error-content';
+
+import './style.scss';
+
+type LicenseErrorKeysType = keyof typeof LICENSE_ERRORS;
+type LicenseErrorValuesType = ( typeof LICENSE_ERRORS )[ LicenseErrorKeysType ];
+
+type Props = {
```

Comment:
I've generally used an interface when defining props and I think that is what is mostly used in our codebases from what I've seen (that could be incorrect). I did some googling and it seems like most people use interface for this due to a "slight" increase in readability and it has a few flexible attributes that types don't have.

All that being said, this is a nitpick and this works fine, the only argument i'd make for interface in this specific scenario is consistency

Reply:
Yes agreed, makes sense 👍